libs/designer/src/lib/ui/CustomNodes/ScopeCardNode.tsx

```diff
@@ -72,6 +74,8 @@ const ScopeCardNode = ({ data, targetPosition = Position.Top, sourcePosition = P
   const nodesMetaData = useNodesMetadata();
   const repetitionName = getRepetitionName(parentRunIndex, scopeId, nodesMetaData, operationsInfo);
   const rootRef = useRef<HTMLDivElement | null>(null);
+  const isTrigger = useMemo(() => metadata?.graphId === 'root' && metadata?.isRoot, [metadata]);
+  const panelTabName = 'SETTINGS';
```

Comment:
actually move this to a constants file somewhere, this is a true constant, not a normal use variable. This is no better than just using the string directly

Reply:
Fixed.